tox.ini

```diff
@@ -3,11 +3,10 @@ name = neurom
 testdeps =
     mock
     nose
-    pathlib2
 
 [tox]
 envlist =
-    {py27,py36}
+    {py36,py37}
```

Comment:
do we need to list all of them, or does tox handle this automatically?

Reply:
Yes, there is no way for tox (locally, on Travis it's another story since it is stated in the yaml file) to know under which python to run those tests except that having this list.